src/translations/en.json

```diff
@@ -1,5 +1,5 @@
 {
-  "ONBOARDING_CONNECT_DD_HEADLINE": "only two things left to do",
+  "ONBOARDING_CONNECT_DD_HEADLINE": "Only two things left to do",
```

Comment:
OK so apparently Zak changed this so no blame on you but it really should _not_ start with a capital letter since it's the second part of a sentence.